cg/meta/demultiplex/status_db_storage_functions.py

```diff
@@ -55,6 +54,23 @@ def store_flow_cell_data_in_status_db(
     store.session.commit()
 
 
+def check_if_samples_have_files(flow_cell: FlowCellDirectoryData) -> None:
+    """Check if all samples have already all the fastq files in the demultiplex directory.
```

Comment:
The doc string is a bit misleading since we are only checking that there is at least one fastq file per sample

Reply:
fixed